src/primitives/segwit.rs

```diff
@@ -9,6 +9,14 @@ use core::fmt;
 
 use crate::primitives::gf32::Fe32;
 
+/// The maximum enforced string length of a segwit address.
+///
+/// The maximum length as specified in BIP-173, this is less than the 1023 character code length.
+/// This limit is based on empirical error-correcting properties, see ["Checksum design"] section.
```

Comment:
In 6ca06b1ec89eb797e5cafd84846a7d7c1e2e37dd:

I'd grep for `, see`. Almost all of the instances should `. See` :).

Reply:
Ha! I literally was looking at someone else's code yesterday thinking the opposite. I'm going to go ahead and trust you, your English tends to be better than mine :)